vm_setup_vars.yml

```diff
@@ -59,6 +61,7 @@ external_network:
     netmask_v4: "{{ baremetal_network_cidr_v4|ipaddr('netmask') }}"
     address_v6: "{{ baremetal_network_cidr_v6|nthhost(1)|default('', true) }}"
     prefix_v6: "{{ baremetal_network_cidr_v6|ipaddr('prefix') }}"
+    dhcp_enabled: "{{ virsh_dhcp_enabled|bool }}"
```

Comment:
Another option is we conditionally define `dhcp_range*` - then we won't need the new metal3-dev-env flag since the `<dhcp` stanza is already skipped when the range is omitted

Reply:
I pushed https://github.com/metal3-io/metal3-dev-env/pull/902 which means we can just set the `dhcp_range` variables to an empty list, which avoids the need for a new variable